index.html

```diff
@@ -1,12 +1,20 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css" /> 
-    <title>Document</title>
-</head>
-<body>
+<html>
+  <head>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+    <link
```

Comment:
I would keep this on the same line and not break it up, unless you're going to do that with the other links.

Over all, it's more of a personal preference but whatever you choose should be consistent through out.

Reply:
So i believe it's prettier formatting this for me, and I have it setup where every time I save it auto formats to it, but I could disable this for now.